src/libcore/python/plugin_v.cpp

```diff
@@ -0,0 +1,13 @@
+#include <mitsuba/core/plugin.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/python/python.h>
+
+MTS_PY_EXPORT(PluginManager) {
+  MTS_PY_IMPORT_TYPES()
+  py::class_<PluginManager, std::unique_ptr<PluginManager, py::nodelete>>(m, "PluginManager")
+    .def_static_method(PluginManager, instance, py::return_value_policy::reference)
+    .def_method(PluginManager, ensure_plugin_loaded, "name"_a)
+    .def_method(PluginManager, loaded_plugins)
+    .def_method(PluginManager, register_python_plugin, "plugin_name"_a)
+    .def("create_object", (ref<Object> (PluginManager::*)(const Properties &props, const Class *class_)) &PluginManager::create_object, "props"_a, "class_"_a, py::return_value_policy::reference);
```

Comment:
I guess the bindings here shouldn't be a one-to-one matching API with the cpp API.

I would just have bindings for `instance()` and `create(..)` for now.
Also, `create(..)` will have to be implemented here (using a `lambda function`) and it will call the `create_object` method once that `Properties` has been created from the dictionary. Something like this:

```cpp
.def("create", [](py::dict &args){
    Properties props = props_from_dict(args);
    return create_object(props, ...);
}, "args"_a)

Reply:
This is the syntax that I want to achieve. However, I can't figure out how to create a ptr to the base class to make a call to the plugin manager with the given props. 
For ex: to create a sensor using the dict in your example for lambda function, how would you obtain `Sensor` class to make a call to `PluginManager::create_object`?

Reply:
I would do it the same way it is done in the `xml.cpp`:
```
PluginManager::instance()->create_object(
        props,
        Class::for_name("Sensor", mitsuba::detail::get_variant<Float, Spectrum>()));
```
Is this what you were looking for?

Reply:
I tried `for_name` with `integrator` only. Thanks for pointing out that I need to capitalize for the class. I will try to use the above suggested fix and submit updates.